content/blog/2020/01/2020-01-10-fosdem-is-coming.adoc

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "FOSDEM 2020 is coming"
+tags:
+- fosdem
+- event
+- community
+- outreachy
+author: olblak
+---
```

Comment:
Add opengraph?

Reply:
What is it used for?

Reply:
For adding images in social media

Reply:
Haaaaa thanks for the information, I am adding one